metricbeat/module/couchbase/bucket/_meta/fields.yml

```diff
@@ -0,0 +1,49 @@
+- name: bucket
+  type: group
+  description: >
+    Couchbase bucket metrics
+  fields:
+    - name: name
+      type: keyword
+      description: >
+        name of the bucket
+    - name: type
+      type: keyword
+      description: >
+        type of the bucket
+    - name: quota.ram
+      type: long
+      description: >
+        Amount of RAM used by the bucket (bytes)
+    - name: quota.raw_ram
+      type: long
+      description: >
+        Amount of raw RAM used by the bucket (bytes)
```

Comment:
so this would be correct:
```
- name: quota.ram.bytes
   format: bytes
   type: long
   description >
     Amount of RAM used by the bucket
```